proteus/mprans/GN_SW2DCV.h

```diff
@@ -909,45 +372,32 @@ class GN_SW2DCV : public GN_SW2DCV_base {
       bar_deltaSqd_kin[i] =
           bar_deltaSqd_kin[i] /
           (csrRowIndeces_DofLoops[i + 1] - csrRowIndeces_DofLoops[i]) / 2.0;
-    }
+    } // i loops ends here
```

Comment:
I like the comment at the end of the loop, it's good practice.

Reply:
Yes, it helps me keep track of where things are in the code B)

Reply:
I would say opinions differ on this practice, but it doesn't really hurt anything and can be useful for the kind of very long loop bodies that we sometimes have in optimized FEM assembly. Any good code editor, however, will easily show you matching brackets or parenthesis, which I trust more than a comment telling me when the loop ends (though the comment nicely records intent for posterity).  In emac, just set your options to highlight matching parenthesis. Of course Python went even further in assuming you have a decent editor by simply using indentation.